x-pack/plugins/infra/public/pages/logs/stream/page_logs_content.tsx

```diff
@@ -24,9 +27,13 @@ import { WithSummary } from '../../../containers/logs/log_summary';
 import { LogViewConfiguration } from '../../../containers/logs/log_view_configuration';
 import { ViewLogInContext } from '../../../containers/logs/view_log_in_context';
 import { WithLogTextviewUrlState } from '../../../containers/logs/with_log_textview';
-import { WithStreamItems } from '../../../containers/logs/with_stream_items';
 import { LogsToolbar } from './page_toolbar';
 import { PageViewLogInContext } from './page_view_log_in_context';
+import { useLogStreamContext } from '../../../containers/logs/log_stream';
+import { datemathToEpochMillis, isValidDatemath } from '../../../utils/datemath';
+
+// FIXME Duplicated from <LogStream />. See where to put this
```

Comment:
Should this be fixed before merge?

Reply:
I think I'm OK with the duplication. We might want the pagination to work differently here than in the `<LogStream />` component.